Move away from kubernetes

enolfc opened the pull request:
Try a simple approach with cron configured with ansible.
Not much different from the k8s approach but for sure will not require a k8s master that takes some resources. Although simpler, it may be harder to scale if this grows too much

gwarf commented on README.md:
```diff
@@ -1,75 +1,67 @@
-the EGI federated cloud.
+Operation of fedcloud integration components for selected providers.
+This is a set of docker containers and an ansible role to operate the
+federation components of the EGI Cloud
```
EGI Cloud Compute?

gwarf commented on README.md:
```diff
@@ -1,75 +1,67 @@
+This is a set of docker containers and an ansible role to operate the
+federation components of the EGI Cloud
 
 ## What this does?
```
Punctuation in title may not be ideal, it's often discouraged (both by the linters and also in some guidelines)

gwarf commented on README.md:
```diff
@@ -1,75 +1,67 @@
 
-The chart will create:
-- a cron job per site that will execute the cloud-info-provider for every 
+The ansible will create:
```
```suggestion
The ansible playbook will create:
```

gwarf commented on README.md:
```diff
@@ -1,75 +1,67 @@
-- a cron job per site that will execute the cloud-info-provider for every 
+The ansible will create:
+- a configuration directory `/etc/fedcloud/vos/<vo>` for every VO
+  that will contain credentials to authenticate into that VO
```
```suggestion
  that will contain credentials to authenticate with that VO
```

Not sure about the proper wording, but into doesn't look great

gwarf commented on README.md:
```diff
@@ -1,75 +1,67 @@
+  credential info from `/etc/fedcloud/vos/<vo>` and sends information
+  to the AMS queue
+- a cron job per site that will execute the cloud-info-provider for every
   vo supporteb by the site and push the information to the AMS in order
```
```suggestion
  vo supported by the site and push the information to the AMS in order
```

gwarf commented on README.md:
````diff
@@ -1,75 +1,67 @@
+## Deploying the chart
 
-helm install -f sites.yaml -f secrets.yaml --name fedcloud fedcloud-ops
+```
````
```suggestion
```sh
```

gwarf commented on README.md:
````diff
@@ -1,75 +1,67 @@
+The `sites` directory contains a YAML file per site to configure following this
+format:
 
 ```
````
```suggestion
```yaml
```

gwarf commented on README.md:
````diff
@@ -1,75 +1,67 @@
+The role expects the following variables to be defined:
+
+- `vos` a map that contains entry for each VO with the Check-in credentials:
+  ```
````
```suggestion
  ```yaml
```

gwarf commented on roles/fedcloud-catchall-ops/tasks/cloud-info.yaml:
```diff
@@ -0,0 +1,52 @@
+    cron_file: "cloud-info-{{ site.gocdb }}"
+  with_fileglob:
+  - "sites/*.yaml"
+
+
```
```suggestion
```

gwarf reviewed (changes requested):
Thanks!
Can't say much on the content itself, I trust it will work or be fixed :)
Added a few cosmetic remarks.

enolfc commented:
I have fixed the README.md and updated sites configuration. 
As I get the service account enrolled into more VOs these will be further populated, but that will come in other PRs

gwarf reviewed (approved):
LGTM!
